homeassistant/components/cloud/strings.json

```diff
@@ -69,6 +69,10 @@
     "reset_bad_custom_domain_configuration": {
       "title": "Custom domain ignored",
       "description": "The DNS configuration for your custom domain ({custom_domains}) is not correct. This domain has now been ignored and will not be used for Home Assistant Cloud. If you want to use this domain, please fix the DNS configuration and restart Home Assistant. If you do not need this anymore, you can remove it from the account page."
+    },
+    "subscription_expired_handler": {
```

Comment:
Maybe just call it `"subscription_expired"`?

Reply:
I thought that first, but we might have other issues later for other subscriptions that have expired, so I did this to avoid making it conflict with a generic term.

Reply:
Ok. `handler` is pretty generic too though. Maybe `"subscription_expired_cloud"`?

Reply:
As we are in the cloud integration here, "cloud" is implied right?

Reply:
Sure, but if we need a suffix it could mean the general cloud subscription.